doc/tags.txt

```diff
@@ -1513,7 +1513,7 @@ v1_6	The XML file now supports comments.
 	Added support for CCB.
         Monitoring refactored
           Most rrd operations now in separate module
-          Groupes multiple attributes in the RRD files (lower overhead)
+          Grouped multiple attributes in the RRD files (lower overhead)
```

Comment:
Should this be `Grouped` or `Groups` (not sure which is more appropriate in the context)?

Reply:
Going with Grouped since it is describing what was done during "monitoring refactored" part